config/config.go

```diff
@@ -80,7 +81,7 @@ func (t *TLS) MakeConfig(serverName string) (*tls.Config, error) {
 		return nil, nil
 	}
 
-	tlsConfig := &tls.Config{}
+	tlsConfig := &tls.Config{} // #nosec G402 -- TLS MinVersion too low - we can't abandon TLS 1.0 as long as Centos 7 is alive.
```

Comment:
CentOS 7 isn't a reason for that. The OpenSSL version in the base system only doesn't support TLS 1.3, TLS 1.2 works just fine there.

Reply:
Does that mean you want to set the min required TLS version to 1.2?

Reply:
But isn't this code being used to create a TLS configuration to access a database or a Redis server? Thus, does, i.e., a MariaDB on CentOS 7 support TLS 1.2?

Reply:
> Does that mean you want to set the min required TLS version to 1.2?

Primarily, I wanted to doubt the claim that "we can't abandon TLS 1.0 as long as Centos 7 is alive".

In general, TLS 1.2 is a reasonable minimal requirement, everything older is better avoided unless there's a very good reason why you would need it (and SSL is a big no right away). And CentOS/RHEL 7, don't seem to be such a reason, the [RHEL 7 documentation already recommends TLS 1.2](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/sec-hardening_tls_configuration#tab-TLS-Protocol_Versions).